public/index.html

```diff
@@ -156,9 +156,16 @@ <h3 class="collection_stats_container"><span class="collection_stats"></span> &n
 						<div id="gifContent" style="text-align: center">
 							<div class="intervalTab">		
 								<label class="inputTitle">Animation Interval</label>
-								<input type="number" name="in" min=".1" step=".1" value="0.5" id="interval">
+								<input type="number" name="in" min=".1" step=".1" value="1" id="interval">
 								<div class="inputControl-addon">sec</div>	
+
+								<label id= "watermarkLabel">
+									<input type="checkbox" id="watermarkOption">
+									Add date and time stamps
```

Comment:
I think you should change this phrase to something like the following (or better):

```suggestion
									Add Timestamp
```